src/Fixer/Alias/RandomApiMigrationFixer.php

```diff
@@ -38,16 +40,24 @@ public function configure(array $configuration = null)
         static $validFunctions = array('rand', 'srand', 'getrandmax');
```

Comment:
why end user is supposed to provide number of arguments ?
we allows to replace only those 3 functions, and they have always same number of arguments. asking user to fill them is pointless. in same time, asking user for more info changing the configure array structure, so BC problem...